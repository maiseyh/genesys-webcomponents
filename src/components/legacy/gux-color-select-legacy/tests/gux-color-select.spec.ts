import { newSpecPage, SpecPage } from '@stencil/core/testing';
import { GuxColorSelectBeta } from '../gux-color-select';

describe('gux-color-select-legacy', () => {
  let page: SpecPage;

  beforeEach(async () => {
    page = await newSpecPage({
      components: [GuxColorSelectBeta],
      html: `<gux-color-select-legacy></gux-color-select-legacy>`,
      language: 'en'
    });
  });

  it('should build', () => {
    expect(page.rootInstance).toBeInstanceOf(GuxColorSelectBeta);
  });

  describe('#render', () => {
    it('should render as expected', () => {
      expect(page.root).toMatchSnapshot();
    });
  });
});
