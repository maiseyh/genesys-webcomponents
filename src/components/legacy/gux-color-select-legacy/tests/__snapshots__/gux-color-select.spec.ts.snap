// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gux-color-select-legacy #render should render as expected 1`] = `
<gux-color-select-legacy>
  <!---->
  <div>
    <div class="gux-color-matrix">
      <gux-color-option-legacy value="#203B73"></gux-color-option-legacy>
      <gux-color-option-legacy value="#75A8FF"></gux-color-option-legacy>
      <gux-color-option-legacy value="#8452CF"></gux-color-option-legacy>
      <gux-color-option-legacy value="#1DA8B3"></gux-color-option-legacy>
      <gux-color-option-legacy value="#B5B5EB"></gux-color-option-legacy>
      <gux-color-option-legacy value="#CC3EBE"></gux-color-option-legacy>
      <gux-color-option-legacy value="#5E6782"></gux-color-option-legacy>
      <gux-color-option-legacy value="#FF8FDD"></gux-color-option-legacy>
      <gux-color-option-legacy value="#868C1E"></gux-color-option-legacy>
      <gux-color-option-legacy value="#DDD933"></gux-color-option-legacy>
    </div>
  </div>
</gux-color-select-legacy>
`;
